src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -56,6 +57,12 @@
         "double_quote", name="quoted_literal", type="literal", trim_chars=('"',)
     ),
     StructKeywordSegment=KeywordSegment.make("struct", name="struct"),
+    StartAngleBracketSegment=SymbolSegment.make(
+        "<", name="start_angle_bracket", type="start_angle_bracket"
```

Comment:
Referring to these as `open_angle_bracket` and `close_angle_bracket` is more common in what I've read but it's not a deal breaker!